.travis.yml

```diff
@@ -41,7 +41,8 @@ install:
     - wget https://raw.githubusercontent.com/pelson/Obvious-CI/master/bootstrap-obvious-ci-and-miniconda.py
     - python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 2 --without-obvci && source ~/miniconda/bin/activate root
     - conda config --set show_channel_urls True
-    - conda install -c pelson/channel/development --yes obvious-ci --force
+    - conda install obvious-ci --yes -c conda-forge
+
     - conda config --add channels ioos
     - python scripts/expand_source.py
```

Comment:
And it is back... I guess that encapsulating `expand_source.py` in a conda-execute call is the only way to ensure it will be called properly inside AppVeyor!

PS: I also had to remove cygwin git ([again](https://github.com/ioos/conda-recipes/commit/79ff4f8f8992deae2c5e66f04f564c6788ed7c2e)) to get things working.